runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java

```diff
@@ -333,4 +333,14 @@ default boolean showPickpocketStun()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "showCombatTimer",
+		name = "Combat logout timer",
+		description = "Configures whether combat logout timer is displayed"
```

Comment:
Maybe this description could be slightly improved? Judging by the name I thought this timer was for combat idle logout (20 minutes AFK) rather than combat cooldown logout (`You can't log out until 10 seconds after the end of combat.`)